src/testData/data.js

```diff
@@ -0,0 +1,90 @@
+// Import data object to simulate response from the backend
+// Feel free to modify the values of the data for testing
+// Do NOT change the structure or namings of the objects without letting everyone know
+
+export const data = {
+  users: [
```

Comment:
Use this data to implement any frontend work that relies on a backend until we get that running.
We are aiming for a very similar so integration will be easier